homeassistant/components/wled/config_flow.py

```diff
@@ -18,12 +19,14 @@ class WLEDFlowHandler(ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
-    async def async_step_user(self, user_input: ConfigType | None = None) -> FlowResult:
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
         """Handle a flow initiated by the user."""
         return await self._handle_config_flow(user_input)
 
     async def async_step_zeroconf(
-        self, discovery_info: ConfigType | None = None
+        self, discovery_info: dict[str, Any] | None = None
```

Comment:
I think `discovery_info` can't be `None`.
https://github.com/home-assistant/core/blob/b0eb2afa65eefd7b428fc9a8085ee422cac4c561/homeassistant/config_entries.py#L1288-L1290
```suggestion
        self, discovery_info: DiscoveryInfoType
```